config/filesystems.php

```diff
@@ -50,6 +50,7 @@
 
         'public' => [
             'driver' => 'local',
+            // @TODO: should we change to storage_path('app/public')?
```

Comment:
This is an interesting, but subtle difference. 

`public_path('storage')` points to `/public/storage/`

`storage_path('app/public')` points to `/storage/app/public/`

Reply:
Oooo interesting - seems safer to keep the same for now?

Reply:
Ya, agreed. I might just make a ticket on the Platforms board to discuss the merits.

Reply:
Ticket created! <https://www.pivotaltracker.com/story/show/174198565>